### ISSUE TITLE
Translation file generation error

### ISSUE THREAD

**Issue by yaroslaw74** [infograf768/j4localise#53]
In some cases, when generating a translation file, the first line gets into the "Not in reference" section. The same thing happens when you translation a large file and periodically save the translated strings. At some point, the message "You are maintaining 1 string that does not exist in the reference language." in this case, if you open a translation text file, the "Not in reference" section appears in it and the first line of the translation is transferred there, and the file itself starts from the second line.

**Comment by infograf768** [comment]
Please post example of the file concerned.
1. The original English version
2. The saved translation.

**Comment by yaroslaw74** [comment]
[en-GB.com_sppagebuilder.zip](https://github.com/infograf768/j4localise/files/8114569/en-GB.com_sppagebuilder.zip)
[ru-RU.com_sppagebuilder.zip](https://github.com/infograf768/j4localise/files/8114573/ru-RU.com_sppagebuilder.zip)

**Comment by infograf768** [comment]
I found a way to solve this, but it implies modifying the head of the en-GB file, i.e. adding 3 empty commented lines.
(I alpha ordered the strings but it has no impact on result, just cleaner.)
```
;
;
;

COM_SPPAGEBUILDER_ERROR_CHECKED_IN="You are not permitted to edit this page because one more user is already checkid-in."
COM_SPPAGEBUILDER_ERROR_EDIT_PERMISSION="You are not permitted to edit this page."
COM_SPPAGEBUILDER_ERROR_PAGE_NOT_FOUND="Page Not Found"
COM_SPPAGEBUILDER_INVALID_CAPTCHA="Invalid Recaptcha"
COM_SPPAGEBUILDER_NO_ITEMS_FOUND="No item found!"
COM_SPPAGEBUILDER_PAGE_CHECKED_OUT="Checked Out"
COM_SPPAGEBUILDER_PAGE_EDIT="Edit Page"
COM_SPPAGEBUILDER_PAGE_SAVE_SUCCESS="Page successfully saved."
COM_SPPAGEBUILDER_PAGE_TITLE_REQUIRED="Page title is required."
COM_SPPAGEBUILDER_RECAPTCHA_NOT_INSTALLED="Please make sure that, re-captcha pluging is enabled"
```
@Valc 

Any idea how to cope with this in code?

**Comment by yaroslaw74** [comment]
Of course, this is also possible, but if the component is updated, through automatic updating, then all changes will be lost. It is extremely inconvenient to carry out such an operation every time.

**Comment by Valc** [comment]
@infograf768 
I will take a look ASAP this weekend.
